aliases/available/git.aliases.bash

```diff
@@ -42,6 +42,7 @@ alias gca='git commit -v -a'
 alias gcm='git commit -v -m'
 alias gcam="git commit -v -am"
 alias gci='git commit --interactive'
+alias gcam='git commit -a -m'
```

Comment:
this looks duplicated on [line 43](https://github.com/Bash-it/bash-it/pull/1237/files#diff-621c6c505c38de5b4f2f4385432041e5R43)

Reply:
true. This one I added over one year ago. I will remove it.